indexFormat/implementation.md

```diff
@@ -1,77 +1,67 @@
 # Building an LSIF exporter
 
-By building an LSIF (Language Server Index Format) exporter for your language server, users can use [Rich Code Navigation](https://code.visualstudio.com/blogs/2018/12/04/rich-navigation) on their pull requests.
+With an LSIF (Language Server Index Format) exporter for your programming language of choice, you can use [Rich Code Navigation](https://code.visualstudio.com/blogs/2018/12/04/rich-navigation) on pull requests inside Visual Studio and Visual Studio Code.
```

Comment:
It might be meaningful to mention here that these are pull requests where you don't have the code locally.